lib/get_value.m

```diff
@@ -0,0 +1,5 @@
+function value = get_value(value)
```

Comment:
Perhaps it would be worth adding a brief comment describing this function's purpose.